hypha/static_src/javascript/file-uploads.js

```diff
@@ -1,26 +1,30 @@
-jQuery(function ($) {
+// We use htmx.onLoad() so it will initilise file uploads in htmx dialogs.
+htmx.onLoad(function () {
   // Initialize django-file-form
-  $("form")
-    .get()
-    .forEach(function (form) {
-      // Prevent initilising it multiple times and run it for forms
-      // that have a `form_id` field added by django-file-form.
-      if (!form.initUploadFieldsDone && form.querySelector("[name=form_id]")) {
-        init(form);
-        form.initUploadFieldsDone = true;
-      }
-    });
+  document.querySelectorAll("form").forEach(function (form) {
+    // Prevent initializing it multiple times and run it for forms
+    // that have a `form_id` field added by django-file-form.
+    if (!form.initUploadFieldsDone && form.querySelector("[name=form_id]")) {
+      init(form);
+      form.initUploadFieldsDone = true;
+    }
+  });
 
   /**
    * Initialize django-file-form for a form.
    * @param {object} form The form to initialize.
    */
   function init(form) {
-    if ($(".form__group--file").length) {
+    if (document.querySelectorAll(".form__group--file").length) {
       window.initUploadFields(form);
 
       // Hide wrapper elements for hidden inputs added by django-file-form
-      $("input[type=hidden]").closest(".form__group").hide();
+      document.querySelectorAll("input[type=hidden]").forEach(function (input) {
+        const closestFormGroup = input.closest(".form__group");
+        if (closestFormGroup) {
+          closestFormGroup.style.display = "none";
+        }
+      });
```

Comment:
Do not think this part is needed anymore. Did not find a place where it had any effect.